daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -691,7 +691,7 @@ object Transaction {
       dependsOnTime: Boolean,
       nodeSeeds: ImmArray[(NodeId, crypto.Hash)],
       globalKeyMapping: Map[GlobalKey, Option[Value.ContractId]],
-      processedDisclosedContracts: ImmArray[ProcessedDisclosedContract],
+      diclosedEvents: ImmArray[Node.Create],
```

Comment:
typo 
```suggestion
      disclosedEvents: ImmArray[Node.Create],
```
(plus a few others)